### PR TITLE
feat(Rest): add property 'endpoint'

### PR DESCRIPTION
**Pull request by doktordirk**
makes life easier if Rest knows which endpoint it is for

### CODE REVIEW

File: src/rest.js [hunk 1]
```diff
@@ -15,9 +15,11 @@ export class Rest {
    * Inject the httpClient to use for requests.
    *
    * @param {HttpClient} httpClient
+   * @param {{string}} [endpoint]
```
Comment (RWOverdijk) [inline src/rest.js]: {string}
Also align httpClient and [endpoint] with spaces

File: src/rest.js [hunk 2]
```diff
@@ -15,9 +15,11 @@ export class Rest {
...
    */
-  constructor(httpClient) {
+  constructor(httpClient, endpoint) {
     this.client = httpClient;
+    this.endpoint = endpoint;
```
Comment (RWOverdijk) [inline src/rest.js]: Align consecutive assignments

**Comment by doktordirk** [comment]
@RWOverdijk that

File: doc/rest.md [hunk 3]
```diff
@@ -6,11 +6,27 @@ import {Rest} from 'spoonx/aurelia-api';
 
 ----------
 
-## .request(method, path[, body][, options])
+## Properties
```
Comment (RWOverdijk) [inline doc/rest.md]: Please direct these changes to [the new docs](http://aurelia-api.spoonx.org/api_rest.html) :)

**Comment by RWOverdijk** [comment]
:+1: if we move the docs. Perhaps also remove the docs dir and add a README in the dir that links to the documentation?

File: doc/rest.md (hunk 3 above)
Comment (RWOverdijk) [inline doc/rest.md]: These changes also require removing spoonx/ from the examples. I'll move it, if you remove docs and add a readme that points here? :)

**Comment by doktordirk** [comment]
pretty sure i removed all spoonx/ in the rename PR. my plan wa first to merge this and the recheck the renaming.
explain me that docs thing on gitter
ps: what about the changelog in docs?

**Comment by doktordirk** [comment]
pps: mind merging this and go on from there?

**Pull request merged**